serving/src/main/resources/application.yml

```diff
@@ -36,11 +36,11 @@ feast:
         # to download the batch features.
         # For example: gs://mybucket/myprefix
         # Please omit the trailing slash in the URI.
-        staging-location: gs://mybucket/myprefix
+        staging_location: gs://mybucket/myprefix
         # Retry options for BigQuery retrieval jobs
-        bigquery-initial-retry-delay-secs: 1
+        bigquery_initial_retry_delay_secs: 1
```

Comment:
Should the `bigquery_` be removed?
https://github.com/gojek/feast/blob/master/protos/feast/core/Store.proto#L124

Reply:
and `secs` to `seconds`

Reply:
yes, sorry. Just noticed

Reply:
Fixed, could you check?